sdks/java/container/build.gradle

```diff
@@ -54,17 +55,6 @@ dependencies {
   dockerDependency library.java.kafka_clients
 }
 
-task copyDockerfileDependencies(type: Copy) {
```

Comment:
Docker tasks were split out into common.gradle

Reply:
Ack, thank you